FluidSynth/FluidSynth.cpp

```diff
@@ -263,6 +281,23 @@ CK_DLL_QUERY( fluidsynth )
     QUERY->add_arg(QUERY, "float[]", "pitches");
     QUERY->add_arg(QUERY, "int", "chan");
 
+    QUERY->add_mfun(QUERY, fluidsynth_setPitchBend, "void", "setPitchBend");
+    QUERY->add_arg(QUERY, "int", "pitchbend");
+
+    QUERY->add_mfun(QUERY, fluidsynth_setPitchBendChan, "void", "setPitchBend");
+    QUERY->add_arg(QUERY, "int", "pitchbend");
+    QUERY->add_arg(QUERY, "int", "chan");
+
+    QUERY->add_mfun(QUERY, fluidsynth_setPitchBend, "void", "resetPitchBend");
+
+    QUERY->add_mfun(QUERY, fluidsynth_setPitchBendChan, "void", "resetPitchBend");
```

Comment:
Should this and the one above it be using 
`fluidsynth_resetPitchBend[Chan]`
?

Reply:
Good catch; fix submitted.